nba_api/stats/library/data.py

```diff
@@ -1137,8 +1154,9 @@
     [2064, "El-Amin", "Khalid", "Khalid El-Amin", False],
     [76653, "Eliason", "Don", "Don Eliason", False],
     [53, "Elie", "Mario", "Mario Elie", False],
+    [1629604, "Elleby", "CJ", "CJ Elleby", True],
     [76655, "Ellefson", "Ray", "Ray Ellefson", False],
-    [1627740, "Ellenson", "Henry", "Henry Ellenson", True],
+    [1627740, "Ellenson", "Henry", "Henry Ellenson", False],
```

Comment:
```suggestion
    [1627740, "Ellenson", "Henry", "Henry Ellenson", True],
```

Ellenson's on the Raptors now.